python/lsst/obs/base/mapper_tests.py

```diff
@@ -170,8 +170,14 @@ def _test_map(self, butlerLocation, dataId):
 
     def test_map(self):
         dataId = self.dataIds['raw']
-        self._test_map(self.mapper.map_raw(dataId), dataId)
-        self._test_map(self.mapper.map("raw", dataId), dataId)
+        location = self.mapper.map_raw(dataId)
+        if not isinstance(location, lsst.daf.persistence.butlerLocation.ButlerComposite):
+            self._test_map(location, dataId)
+        else:
+            self.log.info('ButlerComposite datasets are not tested for mapper functions')
+        location = self.mapper.map("raw", dataId)
```

Comment:
Please add a comment about why this is different from the above: it took me a fair bit of staring to figure it out.

Reply:
Ditto.